elements/nuxeo-suggester/nuxeo-suggester.html

```diff
@@ -210,8 +214,8 @@
         }
 
         paper-input {
-          width: calc(100% - 90px);
-          margin-left: 1.2rem;
+          width: var(--nuxeo-suggester-media-width, calc(100% - 90px));
+          margin-left: var(--nuxeo-suggester-media-margin, 1.2rem);
```

Comment:
Since we might want to add variables to the shorthand `margin` or for the other margins, please include the `-left` in the name:
```suggestion
          margin-left: var(--nuxeo-suggester-media-margin-left, 1.2rem);
```

Reply:
ok it's fixed!